app/presenters/summary/sections/partner_income_benefits_details.rb

```diff
@@ -0,0 +1,41 @@
+require 'laa_crime_schemas'
+
+module Summary
+  module Sections
+    class PartnerIncomeBenefitsDetails < Sections::PaymentDetails
+      def show?
+        true # TODO: Requires adjusting once partner journey in place
```

Comment:
Just thinking you can still set this by feature flag, and if there is a relevant partner or not?

If we don't hide it at least on FF, we can't deploy to prod